Backend/app/common/PropertiesManager.py

```diff
@@ -81,6 +81,8 @@ def _set_attributes(self, config_section: str):
         for key, value in self.config.items(config_section):
             if value == "":
                 value = None
+            if value is None:
```

Comment:
Unify `if value is None:` inside  `if value == "":` in the same `if` statement. Use `or`